CONTRIBUTING.md

```diff
@@ -188,6 +188,24 @@ Follow the example for any of the other pathway reconstruction algorithm.
 First pull the image `<username>/local-neighborhood` from Docker Hub.
 Then build the Docker image using the `Dockerfile` that was completed in Step 2.
 
+Modify Generate Inputs:
+
+Purpose: Update the references and expected files for the new algorithm.
+
+Instructions:
+1. Append the algorithm name to the existing algorithms list.
+2. Include a key-value pair in the algo_exp_file dictionary that links the specific algorithm to its expected network file.
+3. Store the expected network file at /test/GenerateInputs/expected using the naming convention: {algorithm name}-{network file name}-expected.txt.
```

Comment:
To make this as user-friendly as possible for new contributors, I recommend adding a sentence "Obtain the expected network file from..."